tsl/test/expected/compression_merge.out

```diff
@@ -349,8 +349,27 @@ SELECT 12 as expected_number_of_chunks, count(*) as number_of_chunks FROM show_c
                         12 |               12
 (1 row)
 
-SELECT decompress_chunk(i) FROM show_chunks('test6') i;
-             decompress_chunk              
+-- This will generate another 24 chunks
+INSERT INTO test6 
+SELECT t, i, gen_rand_minstd() 
+FROM generate_series('2018-03-03 1:00'::TIMESTAMPTZ, '2018-03-04 0:59', '1 minute') t
+CROSS JOIN generate_series(1, 5, 1) i;
+-- Altering compress chunk time interval will cause us to create 6 chunks from the additional 24 chunks.
+ALTER TABLE test6 set (timescaledb.compress_chunk_time_interval='4 hours');
+SELECT compress_chunk(i, true) FROM show_chunks('test6') i;
+NOTICE:  chunk "_hyper_11_188_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_190_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_192_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_194_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_196_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_198_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_200_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_202_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_204_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_206_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_208_chunk" is already compressed
+NOTICE:  chunk "_hyper_11_210_chunk" is already compressed
```

Comment:
Not sure I follow. The commit message says that you enforce that the `compress_chunk_interval` can only be changed if there are no compressed chunks in the table already, but it does not fail. Did I misunderstand how this was intended to work?

Reply:
Maybe my English is off, I was trying to say this change makes so it can be changed even if there are compressed chunks in the table already.

Reply:
Updated it to make it more clear.